python/lib/database/base.py

```diff
@@ -0,0 +1,5 @@
+from sqlalchemy.orm import DeclarativeBase
+
+
+class Base(DeclarativeBase):
+    pass
```

Comment:
I don't understand the purpose of this class? Or is the plan to fill it up later?